hydrus/tests/unit/test_crud.py

```diff
@@ -33,31 +33,16 @@ def test_crud_get_returns_correct_object(drone_doc_collection_classes, drone_doc
 
 def test_get_for_nested_obj(drone_doc_collection_classes, drone_doc, session, constants):
     """Test CRUD get operation for object that can contain other objects."""
-    API_NAME = constants['API_NAME']
```

Comment:
Do we need to add any new tests to the crud operations for the new schema? You added a few new exceptions, it might be a good idea to add tests for those cases.

Reply:
Yes, I added them